docs/DeveloperGuide.md

```diff
@@ -720,40 +720,212 @@ testers are expected to do more *exploratory* testing.
+    1. Prerequisites: Multiple events in the event list.
+
+    2. Test case: `listEvents s/d`<br>
+       Expected: All events are listed and sorted by their dates from oldest event date to newest event date.
+       A success message is shown in the result display. The top event has the oldest event date.
+
+    3. Test case: `listEvents 123`<br>
+       Expected: Current event list remains the same and no events are sorted. Error message is shown in the result display.
+
+    4. Other incorrect listEvent commands to try: `LISTEVENTS s/d`, `listEvents s/d e`, `listEvents s/n`<br>
+       Expected: Similar to second test case.
+
+    5. `listEvents s/d` lists and sorts all events by their dates even if the event list does not display all events due to the `findEvents` command.
+
+### Tagging Person(s) to an Event
+1. Tagging a person to an Event
+    1. Prerequisites: There must be at least 1 event in the event list and 1 person in the person list.
+
+    2. Whenever a valid tagging is done, all events in the current event list are displayed and the tagged event is updated to reflect the new tagged person.
+
+    3. Test case: `tagEvent 1 p/1`<br>
+       Expected: The first person in the current person list is tagged to the first event in the current event list. A success message is shown in the
+       result display, indicating that the name of the first person has been tagged to the event title of the first event.
+
+    4. Test case: `tagEvent 0 p/1`<br>
+       Expected: No person is tagged to an event. Error message is shown in the result display.
+
+    5. Other incorrect tagEvent commands to try: `tagEvent x p/y` (where either x is larger than the size of the current event list or y is
+       larger than the size of the current person list), `TAGEVENT 1 p/1`, `tagEvent 1 p/x` (where x is the index of a person that is already tagged in the first event
+       in the current event list).<br>
+       Expected: Similar to second test case.
+
+2. Tagging multiple persons to an Event
+    1. Prerequisites: There must be at least 1 event in the event list and more than 1 person in the person list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+    2. Whenever a valid tagging is done, all events in the current event list are displayed and the tagged event is updated to reflect the new tagged persons.
 
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+    3. Test case: `tagEvent 1 p/1 2`<br>
+       Expected: The first and second person in the current person list is tagged to the first event in the current event list. A success message
+       is shown in the result display, indicating that the name of the first and second person has been tagged to the event title of the first event.
```

Comment:
I think the name ... is tagged to the title ... is abit weird, but I understand that's how the success message is displayed, so it's also ok to not change this line
```suggestion
       is shown in the result display, indicating that the first and second person has been tagged to the first event.
```